src/script/hooks/usePushToTalk/usePushToTalk.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {useEffect, useState} from 'react';
+
+import {amplify} from 'amplify';
+
+import {WebAppEvents} from '@wireapp/webapp-events';
+
+import {handleKeyPress} from 'Util/KeyboardUtil';
+
+interface PushToTalk {
+  key: string | null;
+  toggleMute: (shouldMute: boolean) => void;
+  isMuted: () => boolean;
+}
+
+const subscribe = ({key, toggleMute, isMuted}: PushToTalk & {key: string}) => {
```

Comment:
```suggestion
const subscribeToKeyPress = ({key, toggleMute, isMuted}: PushToTalk & {key: string}) => {
```

Let's be a tiny bit more specific here :)